### DISCUSSION TITLE
not actually disabling extension

### DISCUSSION THREAD

**Discussion by IMCGY** [3kh0/ext-remover#43] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
i go onto ingot but i turn the extension off but it dosnt actualy turn it off and i refresh the page and its not disabled

**Comment by chuangjonathan123** [comment]
ye ik ingot shows its disbaled but sites like discord are still blocked by an extension u should create an iboss unblocker

**Comment by ALT4ccount** [comment]
same, it worked at first, and it works for all my friends on the same type of chromebook, just it stopped working for me

**Comment by 3kh0** [comment, marked as answer]
@IMCGY Try using a different methoud.

**Reply by IMCGY** [reply]
I tried but it still isn't actually turning it off

**Reply by 3kh0** [reply]
@IMCGY Try referring to #38 and seeing if that is your issue
